.github/workflows/test.yaml

```diff
@@ -8,5 +8,5 @@ jobs:
     uses: canonical/operator-workflows/.github/workflows/test.yaml@main
     secrets: inherit
     with:
-      self-hosted-runner: true
+      self-hosted-runner: false
```

Comment:
Is there a specific reason for disabling self-hosted runners here? I think we should try to keep them enabled (except that the tests will not run on them) to ensure that we continue to be alpha testers for them.